functions/go/set-namespace/README.md

```diff
@@ -1,28 +0,0 @@
-# Set Namespace Config Function
```

Comment:
We should have a README, but I'm OK adding the doc and example later.

Reply:
https://github.com/GoogleContainerTools/kpt/issues/1245